consensus/polybft/state.go

```diff
@@ -1,152 +1,20 @@
 package polybft
 
 import (
-	"bytes"
-	"encoding/binary"
-	"encoding/json"
-	"errors"
-	"fmt"
-	"math/big"
-	"sort"
-
 	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
-	"github.com/0xPolygon/polygon-edge/types"
 	"github.com/hashicorp/go-hclog"
 	bolt "go.etcd.io/bbolt"
 
 	"github.com/umbracle/ethgo"
 )
 
-/*
-The client has a boltDB backed state store. The schema as of looks as follows:
-
-state sync events/
-|--> stateSyncEvent.Id -> *StateSyncEvent (json marshalled)
-
-commitments/
-|--> commitment.Message.ToIndex -> *CommitmentMessageSigned (json marshalled)
-
-stateSyncProofs/
-|--> stateSyncProof.StateSync.Id -> *StateSyncProof (json marshalled)
-
-epochs/
-|--> epochNumber
-	|--> hash -> []*MessageSignatures (json marshalled)
-
-validatorSnapshots/
-|--> epochNumber -> *AccountSet (json marshalled)
-
-exit events/
-|--> (id+epoch+blockNumber) -> *ExitEvent (json marshalled)
-
-proposer snapshot/
-|--> staticKey - only current one snapshot is preserved -> *ProposerSnapshot (json marshalled)
-*/
-
 var (
 	// ABI
 	stateTransferEventABI = contractsapi.StateSender.Abi.Events["StateSynced"]
```

Comment:
I would move this to StateSync store.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1055/commits/744c97ae877890d400757a57cdfc3b71611ae49e